presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestingDockerizedPostgreSqlServer.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.postgresql;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.testing.docker.DockerContainer;
+
+import java.io.Closeable;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static java.lang.String.format;
+
+public class TestingDockerizedPostgreSqlServer
```

Comment:
```suggestion
public class TestingPostgreSqlServer
```

-- like we have in `TestingSqlServer`

I know we used to use `io.airlift.testing.postgresql.TestingPostgreSqlServer`, but we no longer do after this commit.